pages/index.js

```diff
@@ -161,9 +164,19 @@ export default class Index extends React.PureComponent {
         scrollY: 0,
     };
 
+    propTypes = {
+        animationBackground1: PropTypes.array.isRequired,
+    };
+
     mountainSlideScrollDividend = 50;
     citySlideScrollDividend = 30;
 
+    static async getInitialProps() {
+        const response = await fetch('http://127.0.0.1:8001/.json');
```

Comment:
@tomsvogel How should we move that into some kind of configuration? Using environment variables? What is your favourite approach here?